test/README.md

```diff
@@ -0,0 +1,19 @@
+# Fleet HTTP Client - TESTS
+
+Google Tests
+
+## Requirements
+
+[gtest](https://github.com/google/googletest)
```

Comment:
missing system requirements like port usage

Reply:
added warning about connecting to port 8080